src/components/Common/GetInTouch.js

```diff
@@ -13,8 +12,8 @@ const RowLayout = styled(Row)`
   `}
 `
 
-const GetInTouch = ({ title, contactText }) => (
-  <GreyBackground>
+const GetInTouch = ({ title, contactText, ctaText = 'Get in touch' }) => (
+  <div>
```

Comment:
I dont understand. I see `Get in touch` already in the contact page. has something been deleted from contentful? If that's the case the solution is to make the field mandatory in Contentful